packages/block-editor/src/store/actions.js

```diff
@@ -199,6 +199,17 @@ export function clearSelectedBlock() {
 	};
 }
 
+/**
+ * Returns an action object used in restoring the previously cleared selected blocks.
```

Comment:
Please note that there is a slight difference between the function name and its description. It would more accurate to have:

```js
export function restoreBlockSelection() {
	return {
		type: 'RESTORE_BLOCK_SELECTION',
	};
}
```

I see it follows `clearSelectedBlock` so I'm not sure what's the best way to handle it as changing both would require to keep an alias for the existing method to keep it backward compatible ... 🤷‍♂ @youknowriad - I guess it's fine to keep it singular, right?

Reply:
I like your suggestion about the new name, but I think it is better because it is memorable to stick with restoreSelectedBlock as it is clearly related to restoreSelectedBlock. So we either rename both (and create aliases) or we keep them like this.

Personally I don't like adding empty code with aliases, but that is only pure preference as I also value clarity in naming.